app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/accounts/homepage/HomePageController.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.accounts.homepage
+
+import com.google.inject.Inject
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.{ErrorHandler, ViewConfig}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.SessionUpdates
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.{AuthenticatedAction, RequestWithSessionData, SessionDataAction, WithAuthAndSessionDataAction}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.JourneyStatus.Subscribed
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.{SessionData, UserType}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.repos.SessionStore
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.util.{Logging, toFuture}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.views
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class HomePageController @Inject()(
+                                val authenticatedAction: AuthenticatedAction,
+                                val sessionDataAction: SessionDataAction,
+                                errorHandler: ErrorHandler,
+                                sessionStore: SessionStore,
+                                cc: MessagesControllerComponents,
+                                manageYourDetailsPage: views.html.account.manage_your_details,
+                                homePage: views.html.account.home,
+                                privateBetaHomePage: views.html.account.home_private_beta,
+                                detailUpdatedPage: views.html.account.details_updated,
+                                signedOutPage: views.html.account.signed_out
+                              )(implicit viewConfig: ViewConfig, ec: ExecutionContext)
```

Comment:
apply the formatting - this block of parameters should be two spaces from the left I think

Reply:
and done - I never figured out why the formatter wasn't doing anything on compile